pom.xml

```diff
@@ -56,7 +56,7 @@
     </distributionManagement>
  
     <properties>
-        <jdkVersion>1.7</jdkVersion>
+        <jdkVersion>7</jdkVersion>
```

Comment:
What is the reason for this change? AFAIK, only a Java 9 compiler or later could possibly understand it. 1.7 should be correct, although it might be reasonable to bump it to 1.8.

Reply:
Using '--release' switch for the compiler, which accepts int numbers (7, 8, ...) only. While this brings the requirement on using JDK 11 to build the project, the benefit is that this should guarantee compilation against APIs from the particular version (ie compare to bootstrap settings in Ant)